### PR TITLE
[Fix doc example] - OpenAIGPTDoubleHeadsModel

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

This line fails

https://github.com/huggingface/transformers/blob/7b83feb50a8965e9d8f13b6c4042239710b97c76/src/transformers/models/openai/modeling_openai.py#L690

Should change from `lm_logits` to `logits` (See `OpenAIGPTDoubleHeadsModelOutput`)

## Who can review

@patrickvonplaten, @LysandreJik

### CODE REVIEW

**Review by patil-suraj — APPROVED** [review]
Good catch! Thanks for the fix!
